NEWS.md

```diff
@@ -0,0 +1,4 @@
+# b3doc 0.1.0
+
+* New function `update_frontmatter()` updates the front matter and replaces the logo of a Markdown file on disk (#1, #3, #6).
```

Comment:
Just learned this myself, but please write as:

> New `update_frontmatter()` ...

Not

> New function `update_frontmatter()` ...

https://style.tidyverse.org/news.html#common-patterns